cryptol-saw-core/src/Verifier/SAW/Cryptol.hs

```diff
@@ -90,6 +104,9 @@ import Verifier.SAW.SharedTerm
 import Verifier.SAW.Simulator.MonadLazy (force)
 import Verifier.SAW.TypedAST (mkSort, FieldName, LocalName)
 
+-- local modules:
+import Verifier.SAW.Cryptol.Panic
+
 import GHC.Stack
```

Comment:
Since you are rearranging the modules here, it would be worth moving this `GHC.Stack` import to the top (within the group of modules beginning with `import Control.Monad (...)`) so that it is not confused for a local module.

Reply:
will do.